deeppavlov/core/common/paths.py

```diff
@@ -11,72 +11,40 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import json
+import os
 import shutil
 from pathlib import Path
 
 
-deeppavlov_root = ''
-root_path = Path(__file__).resolve().parent.parent.parent.parent
-default_settings_path = root_path / 'utils' / 'settings' / '.default'
+_root_path = Path(__file__).resolve().parents[3]
+_default_settings_path: Path = _root_path / 'utils' / 'settings'
+_settings_path = Path(os.getenv('DP_SETTINGS_PATH', _default_settings_path)).expanduser().resolve()
 
 
 def get_settings_path() -> Path:
-    """Returns DeepPavlov settings folder absolute path."""
```

Comment:
Please return docstring. We are missing it badly =((

Reply:
Done